package.json

```diff
@@ -36,7 +36,9 @@
   },
   "devDependencies": {
     "broadway": "~0.3.6",
+    "chai": "^4.2.0",
```

Comment:
vows framework is not supported in modern IDEs, it's a hassle to keep supporting it. Building new stuff in it makes no sense.